include/libdnf/rpm/package.hpp

```diff
@@ -0,0 +1,287 @@
+    ReldepList get_obsoletes() const;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.prereq_ignoreinst
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_prereq_ignoreinst(DnfPackage * pkg)
+    ReldepList get_prereq_ignoreinst() const;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.regular_requires
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_regular_requires(DnfPackage * pkg)
+    ReldepList get_regular_requires() const;
+
+    // WEAK DEPENDENCIES
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.recommends
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_recommends(DnfPackage * pkg)
+    ReldepList get_recommends() const;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.suggests
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_suggests(DnfPackage * pkg)
+    ReldepList get_suggests() const;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.enhances
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_enhances(DnfPackage * pkg)
+    ReldepList get_enhances() const;
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.supplements
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_supplements(DnfPackage * pkg)
+    ReldepList get_supplements() const;
+
+    // REPODATA
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.baseurl
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_baseurl(DnfPackage * pkg)
+    std::string get_baseurl();
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.location
+    /// @replaces dnf:dnf/package.py:attribute:Package.relativepath
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_get_location(DnfPackage * pkg)
+    std::string get_location();
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.chksum
+    /// @replaces libdnf:libdnf/hy-package-private.hpp:function:dnf_package_get_chksum(DnfPackage *pkg, int *type)
+    Chksum get_chksum();
+
+    /// @replaces dnf:dnf/package.py:attribute:Package.hdr_chksum
+    /// @replaces libdnf:libdnf/hy-package-private.hpp:function:dnf_package_get_hdr_chksum(DnfPackage *pkg, int *type)
+    Chksum get_hdr_chksum();
+
+    /// TODO get_changelogs - requires changelog
+    /// @replaces dnf:dnf/package.py:attribute:Package.changelogs
+    /// @replaces libdnf:libdnf/hy-package-private.hpp:function:dnf_package_get_changelogs(DnfPackage * pkg)
+    // void get_changelogs() const;
+
+    // SYSTEM
+
+    /// @replaces libdnf:libdnf/hy-package.h:function:dnf_package_installed(DnfPackage * pkg)
+    /// @replaces dnf:dnf/package.py:attribute:Package.installed
+    bool is_installed() const;
+
+    /// TODO is_local
+    /// @replaces dnf:dnf/package.py:method:Package.localPkg(self)
+    /// @replaces libdnf:libdnf/dnf-package.h:function:dnf_package_is_local(DnfPackage * pkg)
+    //bool is_local() const;
```

Comment:
local vs @Commandline repo? Maybe the relation could be explicitly documented.

Reply:
Local is not only commandline repository, but also packages from local repository.